apps/tasks/lib/controllers/authentication_controller.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/foundation.dart';
+import 'package:tasks/services/auth_service.dart';
+
+/// A Controller for providing and updating the current authentication status
+class AuthenticationController extends ChangeNotifier {
+  final AuthService service = AuthService();
+
+  get isInitialized => service.isInitialized;
+
+  get hasTriedTokens => service.hasTriedTokens;
+
+  AuthenticationController() {
+    tokenLogin();
+  }
+
+  Future<void> tokenLogin() async {
+    await Future.wait([
+      service.tokenLogin(),
+      // So the loading on the LandingScreen doesn't just blink
+      Future.delayed(const Duration(seconds: 1)),
```

Comment:
No, that takes like 50ms-300ms, I guess, but then the Loading indicator just blinks in that duration, and it would be more understandable for the user to show it for a second. So they know what is happening.